lib/CustomFields/components/FieldAccordion/edit-fields/shared-fields/OptionsField/tests/OptionsField-test.js

```diff
@@ -115,11 +115,6 @@ describe('OptionsField', () => {
     describe('and reaching a max number of options', () => {
       beforeEach(async () => {
         await optionsField.addOption();
-        a11yResults = await axe.run();

```

Comment:
@BogdanDenis was this giving you some trouble?

Reply:
Yes, sometimes tests would fail on this step. And we actually shouldn't need a11y checks here because when maximum number of options is reached we're hiding a single button and nothing else - so there should be any a11y issues

Reply:
Thanks @BogdanDenis . That makes sense. I ran into similar issues with axe too.